src/hb-ot-font.cc

```diff
@@ -180,7 +181,11 @@ hb_ot_get_glyph_extents (hb_font_t *font,
 			 void *user_data HB_UNUSED)
 {
   const hb_ot_face_data_t *ot_face = (const hb_ot_face_data_t *) font_data;
-  bool ret = ot_face->glyf->get_extents (glyph, extents);
+  bool ret = false;
+  if (ot_face->sbix->has_data ())
```

Comment:
No has_data check needed!

You're not learning to embrace the nul objects.

Reply:
Hehehe, right